src/AppBundle/Resources/views/Person/show.json.twig

```diff
@@ -0,0 +1,62 @@
+
+   {
+        
+   "@context": "http://schema.org",
+   "@type": "Person",
+
+   "url": "{{ url('person_show', {'id': person.id}) }}",
+   "familyName": "{{ person.firstName }}",
+   "givenName": "{{ person.lastName }}",
+   {% if person.title %}
+   "jobTitle": "{{ person.title }}",
+   {% endif %}
+   "birthDate": "{{ person.dob }}",
+   "deathDate": "{{ person.dod }}",
+   {% if person.gender == 'F' %}
+   "gender": "female",         
+   {% elseif person.gender == 'M' %}
+   "gender": "male",        
+   {% else %}
+   "gender": "unknown", 
+   {% endif %}
+       
+   "birthPlace": {
+   {% if person.cityOfBirth and person.cityOfBirth.geonameid != 0 %}
+        "@type": "Place",
+        "url": "http://sws.geonames.org/{{ person.cityOfBirth.geonameid }}/about.rdf"
+   {% endif %}
+       },
+       
+   "deathPlace": {   
+   {% if person.cityOfDeath and person.cityOfDeath.geonameid != 0 %}
+        "@type": "Place",
+        "url": "http://sws.geonames.org/{{ person.cityOfDeath.geonameid }}/about.rdf"
+   {% endif %}
+       },
+       
+   {% for tf in person.titleRoles %}
+            
+       {% if tf.role.name == 'Author' %}
```

Comment:
Again, these should be lists. If someone was an author of multiple works, only the first or last one would be picked up by the processor, for example.